Source/WebCore/html/HTMLDocument.h

```diff
@@ -67,7 +67,7 @@ class HTMLDocument : public Document {
 
 inline Ref<HTMLDocument> HTMLDocument::create(LocalFrame* frame, const Settings& settings, const URL& url, ScriptExecutionContextIdentifier identifier)
 {
-    auto document = adoptRef(*new HTMLDocument(frame, settings, url, identifier, { DocumentClass::HTML }, 0));
+    auto document = adoptRef(*new HTMLDocument(frame, settings, url, identifier, { DocumentClass::HTML }, { }));
```

Comment:
nit: seems we can simply omit the last parameter.

Reply:
Good point.